DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/textSearch/fTSQuerySearchBoosting/metadata.json

```diff
@@ -0,0 +1,118 @@
+{
+    "Slug": "fts-query-search-boosting",
+    "SourceFileName": "FTSQuerySearchBoosting.java",
+    "Title": "FTS Query - Boosting",
+    "DescriptionLines": [
+        "**Full Text Search query** allows searching for documents containing particular term(s) within a block of text.",
+        "",
+        "**Provide a boost value** to the searched terms in order to prioritize the results.",
+        "The higher the boost value, the higher the ranking of its matching document in the results.",
+        "",
+        "Upon query, an auto-index is created on the searched document fields."
+    ],
+
+    "Assets": [
+        {
+            "Title": "Operation in queries - Book",
+            "Type" : "link",
+            "Url"  : "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/9-querying-in-ravendb#operations-in-queries"
+        },
+        {
+            "Title": "Full text searching - Book",
+            "Type" : "link",
+            "Url"  : "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/9-querying-in-ravendb#full-text-searching"
+        }/*,
+        {
+            "Title": "How to search",
+            "Type" : "link",
+            "Url"  : "https://ravendb.net/docs/article-page/latest/java/client-api/session/querying/how-to-use-search"
+        },
+        {
+            "Title": "Querying searching",
+            "Type" : "link",
+            "Url"  : "https://ravendb.net/docs/article-page/latest/java/indexes/querying/searching"
+        }*/,
+        {
+            "Title": "Querying boosting",
+            "Type" : "link",
+            "Url"  : "https://ravendb.net/docs/article-page/latest/java/indexes/querying/boosting"
+        },
+        {
+            "Title": "Indexes boosting",
+            "Type" : "link",
+            "Url"  : "https://ravendb.net/docs/article-page/latest/java/indexes/boosting"
+        }
+        
+    ],
+
+    "Walkthroughs": [
+        {
+            "Title": "Define a query",
+            "Slug": "step-1",
+            "DescriptionLines": [
+                "* Define a query on the 'Employees' collection Using the Session `query()`."
+            ],
+            "Assets": [
+                {
+                    "Title": "Query Overview Demo",
+                    "Type" : "demo",
+                    "Url"  : "/java/queries/query-overview"
+                }
+            ]
+        },
+        {
+            "Title": "Define search criteria",
+            "Slug": "step-2",
+            "DescriptionLines": [
+                "* Use `search()` to search for the specified terms on the specified document field.",
+                "",
+                "* Use `.boost(boostValue)` to set the boost value per term.<br>",
```

Comment:
lets have it without the `.`  - similar the `search()`

`.boost(boostValue)` => `boost(boostValue)`

Reply:
Done